src/_guides/language/language-tour.md

```diff
@@ -2601,7 +2601,7 @@ initializers with commas.
 {% prettify dart %}
 // Initializer list sets instance variables before
 // the constructor body runs.
-Point.fromJson(Map json)
+Point.fromJson(Map<String, num> json)
```

Comment:
This is OK for now, but we're going to need to introduce generics earlier. I'll file an issue.